psl/builtin-connectors/src/mongodb.rs

```diff
@@ -132,4 +132,9 @@ impl Connector for MongoDbDatamodelConnector {
     fn allowed_relation_mode_settings(&self) -> enumflags2::BitFlags<RelationMode> {
         RelationMode::Prisma.into()
     }
+
+    // avoid checking whether the referencing fields in a `@relation` attribute belong to an index during the schema validation
+    fn should_suggest_missing_referencing_fields_indexes(&self) -> bool {
```

Comment:
Do we have a reason to not check this with Mongo? Also the comment here is a bit weird, what does it say? It should also use `///` to be a rustdoc comment.

Reply:
@Jolg42 and @janpio agreed on leaving Mongo's index out of the equation for the time being, as we do not know that much about the performance impact of Mongo's indexes at the moment. They might be dealt with in the future, or maybe never.

Reply:
Can confirm: MongoDB is out of scope for the current project and sprint.